python/pyspark/sql/session.py

```diff
@@ -252,6 +253,22 @@ def newSession(self):
         """
         return self.__class__(self._sc, self._jsparkSession.newSession())
 
+    @since(3.0)
```

Comment:
Let's change this to 2.5

Reply:
@HyukjinKwon are you OK to mark this comment as resolved since we're now targeting `3.0`?

Reply:
Yes, at that time, 2.5 was targeted. Now 3.0 is targeted per https://github.com/apache/spark/commit/9bf397c0e45cb161f3f12f09bd2bf14ff96dc823